### DISCUSSION TITLE
Remote Georgia Tech Student Can't Get Approved for Educational Account

### DISCUSSION THREAD

**Discussion by BigTMiami** [community/community#51270] [GitHub Education]
I am part of the OMSCS Graduate Program.  They generally don't differentiate between remote and on campus.  I don't have any documents that are both Transcripts \ Current and also highlight the remote status.  I think I am exactly the kind of person this account is geared towards.

 Help!

**Comment by samus-aran** [comment]
Hi @BigTMiami 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
